dataclients/kubernetes/definitions/routegroupvalidator.go

```diff
@@ -56,7 +63,10 @@ func (rgv *RouteGroupValidator) filtersValidation(item *RouteGroupItem) error {
 	var errs []error
 	for _, r := range item.Spec.Routes {
 		for _, f := range r.Filters {
-			_, err := eskip.ParseFilters(f)
+			filters, err := eskip.ParseFilters(f)
+			if len(filters) > 1 {
```

Comment:
I think we should check number of filters only when err is nil.

Reply:
Why? I think we should send both errors to decrease roundtrips of fixes

Reply:
Lets add a test case for empty value, maybe eskip.Parse already handles it

Reply:
there is one already

* https://github.com/zalando/skipper/blob/master/dataclients/kubernetes/definitions/testdata/validation/route-with-empty-filter.yaml
* https://github.com/zalando/skipper/blob/master/dataclients/kubernetes/definitions/testdata/validation/route-with-empty-predicate.yaml

Reply:
without validation `ParseFilter/predicate` return no error for empty string